src/modules/InstShowInfo.rb

```diff
@@ -17,7 +17,7 @@ def main
       Yast.import "Label"
     end
 
-    # @param [String] info_file (/info.txt" - Copied from inst media to inst-sys by linuxrc)
+    # @param [String] info_file path to be shown
```

Comment:
is it really info file or beta one? Maybe not worth it to change API name

Reply:
yep, by now I decided to not change the API name